rules/Line.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * This script conflates all linear features using Generic Conflation.
+ */
+
+"use strict";
+
+exports.candidateDistanceSigma = 1.0; // 1.0 * (CE95 + Worst CE95);
+exports.description = "Matches generic lines";
+exports.experimental = false;
+exports.matchThreshold = parseFloat(hoot.get("generic.line.match.threshold"));
+exports.missThreshold = parseFloat(hoot.get("generic.line.miss.threshold"));
+exports.reviewThreshold = parseFloat(hoot.get("generic.line.review.threshold"));
+exports.searchRadius = parseFloat(hoot.get("search.radius.generic.line"));
+exports.tagThreshold = parseFloat(hoot.get("generic.line.tag.threshold"));
+exports.baseFeatureType = "Line";
+exports.geometryType = "line";
+
+var angleHistogramExtractor = new hoot.AngleHistogramExtractor();
+var weightedShapeDistanceExtractor = new hoot.WeightedShapeDistanceExtractor();
+var distanceScoreExtractor = new hoot.DistanceScoreExtractor();
+var lengthScoreExtractor = new hoot.LengthScoreExtractor();
+var sublineMatcher = new hoot.MaximalSublineStringMatcher({
+    "way.matcher.max.angle": hoot.get("generic.line.matcher.max.angle"),
+    "way.subline.matcher": hoot.get("generic.line.subline.matcher")});
+
+/**
+ * Returns true if e is a candidate for a match. Implementing this method is
+ * optional, but may dramatically increase speed if you can cull some features
+ * early on. E.g. no need to check nodes for a polygon to polygon match.
+ *
+ * exports.matchCandidateCriterion takes precedence over this function and must
+ * be commented out before using it.
+ */
+exports.isMatchCandidate = function(map, e)
```

Comment:
Make sure not to try and conflate anything here that can be conflated by one of the custom routines.